components/pd_client/src/lib.rs

```diff
@@ -75,6 +77,21 @@ pub const INVALID_ID: u64 = 0;
 /// creating the PdClient is enough and the PdClient will use this cluster id
 /// all the time.
 pub trait PdClient: Send + Sync {
+    /// Load a list of GlobalConfig
+    fn load_global_config(&self, _list: Vec<String>) -> PdFuture<HashMap<String, String>> {
+        unimplemented!();
+    }
+
+    /// Store a list of GlobalConfig
+    fn store_global_config(&self, _list: HashMap<String, String>) -> PdFuture<()> {
```

Comment:
Why need store?
Only TiDB needs to modify a global variable

Reply:
but we need to implement all methods

Reply:
We definitely do not need to define and/or implement "store_global_config" method in TiKV

cc @crazycs520

Reply:
If it's not needed, we can use `unimplemented!()`.

Reply:
If it's really not needed, it should be removed from the rpc proto...

Reply:
TiKV PD client currently no need to implement `store_global_config`.